packages/discord.js/src/structures/GuildAuditLogsEntry.js

```diff
@@ -391,9 +397,10 @@ class GuildAuditLogsEntry {
     if (target < 110) return Targets.GuildScheduledEvent;
     if (target < 120) return Targets.Thread;
     if (target < 130) return Targets.ApplicationCommand;
-    if (target >= 140 && target < 150) return Targets.AutoModeration;
+    if (target < 140) return Targets.SoundboardSound;
+    if (target >= 140 && target <= 142) return Targets.AutoModeration;
+    if (target >= 143 && target <= 145) return Targets.User;
```

Comment:
We already know the first conditions are true at this point.
Also updated to use `<` to more or less keep the pattern that was being followed on this method.

```suggestion
    if (target < 143) return Targets.AutoModeration;
    if (target < 146) return Targets.User;
```

Reply:
Thanks @almeidx, fixed